src/tests/ftest/checksum/csum_error_logging.py

```diff
@@ -72,7 +72,7 @@ def test_csum_error_logging(self):
                 if not device['tgt_ids']:
                     self.log_step('Skipping device without targets on {}'.format(device['uuid']))
                     continue
-                if device['roles'] and not int(device['role_bits']) & 1:
+                if not int(device['role_bits']) & 1:
```

Comment:
If we don't care about `roles` should we remove it from here too?
https://github.com/daos-stack/daos/blob/8c91a8a138099a3d2164e4582891e5dfd93438f5/src/tests/ftest/checksum/csum_error_logging.py#L65-L71

Reply:
I noticed this place... Since it just checks whether there is an entry for the device [keys], I thought it is ok to keep it for now. If you feel we should remove it, I am fine to remove it.

Reply:
I think if we don't need `roles` for this test, we should remove it. But not a hard blocker.

Reply:
Done.